platform/admin/core/admin-core-logviewer/src/main/webapp/js/components/log-entry.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+var React = require('react')
+var moment = require('moment')
+
+var levels = require('../levels')
+
+var format = function (time) {
+  return moment(time).format('D MMM YYYY, HH:mm:ss')
+}
+
+var styles = function (level) {
```

Comment:
In this situation, all of the styles are very specific and tightly coupled to the components that are contained in their js file. It made it very straightforward to change the style, for example, of log-entry simply by going to that component's js file and modifying its css.

Is there a use case in this situation that we are worried about that warrants adding an additional layer of indirection by adding classes and moving all of the nicely scoped styles into a separate css file?